src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImplTest.java

```diff
@@ -175,4 +187,61 @@ public void getCaseByUPRNNotFound() throws Exception {
 
     assertThrows(CTPException.class, () -> caseSvc.getLatestValidNonHICaseByUPRN(UPRN));
   }
+
+  @Test
+  public void shouldCreateNewCaseRequestPayload() throws Exception {
+    callAndVerifyNewCaseCreated();
+  }
+
+  private void callAndVerifyNewCaseCreated() throws CTPException {
+
+    NewCaseDTO newCaseDTO = getNewCaseDto();
+
+    caseSvc.sendNewCaseEvent(newCaseDTO);
+
+    verify(eventPublisher)
+        .sendEvent(
+            eq(TopicType.NEW_CASE),
+            eq(Source.RESPONDENT_HOME),
+            eq(Channel.RH),
+            sendEventCaptor.capture());
+    NewCasePayloadContent eventPayload = sendEventCaptor.getValue();
+
+    assertEquals(newCaseDTO.getCollectionExerciseId(), eventPayload.getCollectionExerciseId());
+    assertEquals(newCaseDTO.getSchoolId(), eventPayload.getSample().getSchoolId());
+    assertEquals(newCaseDTO.getSchoolName(), eventPayload.getSample().getSchoolName());
+    assertEquals(newCaseDTO.getFirstName(), eventPayload.getSampleSensitive().getFirstName());
+    assertEquals(newCaseDTO.getLastName(), eventPayload.getSampleSensitive().getLastName());
+    assertEquals(
+        newCaseDTO.getChildFirstName(), eventPayload.getSampleSensitive().getChildFirstName());
+    assertEquals(
+        newCaseDTO.getChildMiddleName(), eventPayload.getSampleSensitive().getChildMiddleNames());
+    assertEquals(
+        newCaseDTO.getChildLastName(), eventPayload.getSampleSensitive().getChildLastName());
+    assertEquals(newCaseDTO.getChildDob(), eventPayload.getSampleSensitive().getChildDob());
+    assertEquals(
+        newCaseDTO.getParentMobileNumber(),
+        eventPayload.getSampleSensitive().getParentMobileNumber());
+    assertEquals(
+        newCaseDTO.getParentEmailAddress(),
+        eventPayload.getSampleSensitive().getParentEmailAddress());
+  }
+
+  public NewCaseDTO getNewCaseDto() {
```

Comment:
This is not our normal fixture method. I'm not a fan, but our standard pattern is to create a JSON file and covert it to an Object with `FixtureHelper` eg, like the collectionCase above in this file.